components/button-cards/_category-card/category-card.js

```diff
@@ -2,6 +2,11 @@ window.addEventListener('load', function() {
   const cardRows = document.querySelectorAll('.card-row')
 
   cardRows.ensureCardContentsSameHeightWithinParent({
-    subSelectors: ['.msds-category-card__header-body', '.msds-category-card__content-body']
+    subSelectors: [
+      '.msds-solution-card__header-headline',
+      '.msds-solution-card__body-headline',
+      '.msds-solution-card__body-description',
+      '.msds-solution-card__footer-buttons'
+    ]
   })
 })
```

Comment:
It looks like the category card code is in the solution cards JS file, and the solution cards code is in the category cards JS file